nmdc_runtime/site/export/ncbi_xml.py

```diff
@@ -0,0 +1,295 @@
+                    "Attributes",
+                    children=[
+                        self.set_element(
+                            "Attribute", attributes[key], {"attribute_name": key}
+                        )
+                        for key in sorted(attributes)
+                    ],
+                ),
+            ]
+
+            action = self.set_element(
+                "Action",
+                children=[
+                    self.set_element(
+                        "AddData",
+                        attrib={"target_db": "BioSample"},
+                        children=[
+                            self.set_element(
+                                "Data",
+                                attrib={"content_type": "XML"},
+                                children=[
+                                    self.set_element(
+                                        "XmlContent",
+                                        children=[
+                                            self.set_element(
+                                                "BioSample",
+                                                attrib={"schema_version": "2.0"},
+                                                children=biosample_elements,
+                                            ),
+                                        ],
+                                    ),
+                                ],
+                            ),
+                            self.set_element(
+                                "Identifier",
+                                children=[
+                                    self.set_element(
+                                        "SPUID",
+                                        sample_id_value,
+                                        {"spuid_namespace": org},
+                                    ),
+                                ],
+                            ),
+                        ],
+                    ),
+                ],
+            )
+            self.root.append(action)
+
+    def get_submission_xml(self):
+        self.set_description(
+            email=self.ncbi_submission_metadata.get("email", ""),
+            user=self.ncbi_submission_metadata.get("user", ""),
+            first=self.ncbi_submission_metadata.get("first", ""),
+            last=self.ncbi_submission_metadata.get("last", ""),
+            org=self.ncbi_submission_metadata.get("organization", ""),
+        )
+
+        self.set_bioproject(
+            title=self.ncbi_bioproject_metadata.get("title", ""),
+            project_id=self.ncbi_bioproject_metadata.get("project_id", ""),
+            description=self.ncbi_bioproject_metadata.get("description", ""),
+            data_type=self.ncbi_bioproject_metadata.get("data_type", ""),
+            org=self.ncbi_submission_metadata.get("organization", ""),
+        )
+
+        biosamples_list = self.nmdc_api_client.get_biosamples_part_of_study(
+            self.nmdc_study_id
+        )
```

Comment:
Similar to above, I would prefer to see this list obtained externally and passed to this method.

Reply:
Good point! addressed this by writing logic in an `@op` called `get_biosamples_by_study_id()`.